scripts/eslint.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+
+if which node > /dev/null; then
+    export NVM_DIR="$HOME/.nvm"
+    [ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh"
+
+    cd frontend && yarn && npm run lint:staged
```

Comment:
My concern is yarn will run really slow. Especially for backend developers.
I'm still working on looking for a way to run frontend linter only when frontend files are committed.

Reply:
@ecmadao I think you can set `files` in the `hooks` configuration so that the hook only run for the specified file pattern.

Reply:
Through the existed pre-commit dependency we can remove the bash now